### PR TITLE
LPS-159740: An inconsistency in the behavior about wikiPage of API Explorer

### PR DESCRIPTION
**Pull request by sergiojimcos**
Original PR: https://github.com/liferay-frontend/liferay-portal/pull/2604
Hi guys, I've opened this PR because most of the service changes I've made are related to your components. I'm going to introduced what are the changes that you should reviewed.

**What is new?:**

- Add new method (`getPageByResourcePrimKey`) in `WikiPageServiceImpl` to retrive a wiki page given a WikiPage resource Primary Key:

```
	public WikiPage getPageByResourcePrimKey(long resourcePrimKey)
		throws PortalException {

		WikiPage page = wikiPageLocalService.getPage(resourcePrimKey);

		_wikiPageModelResourcePermission.check(
			getPermissionChecker(), page, ActionKeys.VIEW);

		return page;
	}
```
- Update `WikiPageAttachmentsResourceImpl` using the new method.
- Update `WikiPageAttachmentResourceTest` retrieving the wikiPage resource Primary Key.

**SCENARIO** 

Once I create a wikiPage with attachments when I use getWikiPageWikiPageAttachmentsPage with wikiPageId.

**Note:** The syntax of the new method (`getPageByResourcePrimKey`) would follow the other methods from WikiPageService to maintain the homogeneity and not to cause misinterpretations.

**Before changes**
Response return "No WikiPage exists with the primary key"

**After changes**
Response 200 OK

**Ticket JIRA**
https://issues.liferay.com/browse/LPS-159740

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/wiki/wiki-service/src/main/java/com/liferay/wiki/service/impl/WikiPageServiceImpl.java [hunk 1]
```diff
@@ -549,6 +549,18 @@ public WikiPage getPageByPageId(long pageId) throws PortalException {
 		return page;
 	}
 
+	@Override
+	public WikiPage getPageByResourcePrimKey(long resourcePrimKey)
```
Comment (AliciaGarciaGarcia) [inline modules/apps/wiki/wiki-service/src/main/java/com/liferay/wiki/service/impl/WikiPageServiceImpl.java:553]: getPageByResourcePrimKey > getWikiPage...

it return a wikiPage ... :D

File: modules/apps/wiki/wiki-service/src/main/java/com/liferay/wiki/service/impl/WikiPageServiceImpl.java [hunk 2]
```diff
@@ -549,6 +549,18 @@ public WikiPage getPageByPageId(long pageId) throws PortalException {
...
+		throws PortalException {
+
+		WikiPage page = wikiPageLocalService.getPage(resourcePrimKey);
```
Comment (AliciaGarciaGarcia) [inline modules/apps/wiki/wiki-service/src/main/java/com/liferay/wiki/service/impl/WikiPageServiceImpl.java:556]: same here > wikiPage

**Review by AliciaGarciaGarcia — CHANGES_REQUESTED** [review]

File: modules/apps/wiki/wiki-service/src/main/java/com/liferay/wiki/service/impl/WikiPageServiceImpl.java (hunk 1 above)
Comment (AliciaGarciaGarcia) [reply modules/apps/wiki/wiki-service/src/main/java/com/liferay/wiki/service/impl/WikiPageServiceImpl.java:553]: Oh!, I see that on that class ALL the methods have the same name...
Maybe is better to maintain the same as other methods

**Review by AliciaGarciaGarcia — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
liferay/ci:test:stable

**Comment by AliciaGarciaGarcia** [comment]
ci:test:stable

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:test:stable

**Comment by austinchiang** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:stop

**Comment by AliciaGarciaGarcia** [comment]
ci:test:stable

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**